test/gtest/uct/test_p2p_err.cc

```diff
@@ -109,9 +110,16 @@ class uct_p2p_err_test : public uct_p2p_test {
             }
         }
 
-        EXPECT_GT(num_matched, 0ul) <<
-                        "No error which contains the string '" << error_pattern <<
-                        "' has occurred during the test";
+        if (is_access_error) {
+            EXPECT_EQ(num_matched, 0ul) <<
+                            "Errors which contain the string '" << error_pattern <<
+                            "' have occurred during the test";
```

Comment:
in case of "access error" we should expect error message from UCX, since this is not considered a connection issue, but a local issue

Reply:
even _remote_ access error?

Reply:
yes, when it's reported as error CQE